### PR TITLE
Add environment

### PR DESCRIPTION
**Pull request by echodpp**
Please review the environment file @fdzzzz

### CODE REVIEW

**Comment by fdzzzz** [comment]
Perhaps we can define what the object is in the class. We could also add some hints and comments on the function, especially on ini_board_mine() and respond_query().

Thanks,
Dezhao

File: Environment.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+    def mark_mine(self, mine_node):
+        (x, y) = mine_node
+        self.hidden_board[x][y] = -1
+
+    def respond_query(self, i, j, mark_safe_action=0):
+        if self.board[i][j] == -1:
+            self.uncover((i, j))
+            return False
+        if mark_safe_action == 1:
```
Comment (fdzzzz) [inline Environment.py]: we could use elif for here

File: Environment.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import random
+
+
+class Environment(object):
+    def __init__(self, d, n):
+        self.dim = d
```
Comment (fdzzzz) [inline Environment.py:9]: we could comment d as dimension

**Comment by echodpp** [comment]
just add type hints/ will add some more comment tomorrow if we have time ~ thanks

**Comment by xianxinzhu** [comment]
I can add type hints

**Pull request merged**